cmd/result.go

```diff
@@ -7,29 +7,27 @@ import (
 )
 
 type Result struct {
+	CPULimitActual string
+	CPULimitMax    string
+	DSA            string
 	Err            int
-	Occurrences    []Occurrence
-	Namespace      string
-	Name           string
-	CapsAdded      []Capability
 	ImageName      string
-	CapsDropped    []Capability
```

Comment:
Where did the `Caps...` variables go? Do we now just use: `container.SecurityContext.Capabilities.*` instead?

Reply:
`CapsAdded` and `CapsNotDropped` are no longer used since we have a single occurrence per erroneous cap (and we can use individual capNames). I might have gotten carried away deleting `CapsDropped` but we don't actually use it anywhere at the moment so I'm not sure what its there for